src/core/lv_refr.c

```diff
@@ -610,7 +610,7 @@ static void lv_refr_area_part(lv_draw_ctx_t * draw_ctx)
             lv_draw_rect_dsc_init(&dsc);
             dsc.bg_img_src = disp_refr->bg_img;
             dsc.bg_img_opa = disp_refr->bg_opa;
-            dsc.bg_color = disp_refr->bg_color;
+            dsc.bg_grad.stops[0].color = disp_refr->bg_color;
```

Comment:
Could you revert the occurrences of `.bg_grad.stops[0].color` to `.bg_color` where applicable?